podman/tests/integration/test_container_exec.py

```diff
@@ -49,5 +49,42 @@ def test_container_exec_run_demux(self):
         error_code, output = container.exec_run("ls nonexistent", demux=True)
 
         self.assertEqual(error_code, 1)
-        self.assertEqual(output[0], b'')
+        self.assertEqual(output[0], None)
         self.assertEqual(output[1], b"ls: nonexistent: No such file or directory\n")
+
+    def test_container_exec_run_stream(self):
+        """Test streaming the output from a long running command."""
+        container = self.client.containers.create(self.alpine_image, command=["top"], detach=True)
+        container.start()
+
+        command = []
+        for i in range(3):
+            # We want to sleep so that the lines get  processed seperately
+            command.extend(['echo', str(i), ';', 'sleep', '.1', ';'])
+        command = ['/bin/sh', '-c', ' '.join(command)]
```

Comment:
is there a reason not to have the command written explicitly but with a for loop? I would prefer to have the tests more plain, and I don't think there is a special need for a `for`

Reply:
I personally find the loop easier to read as I think it better highlights the iterative nature of the expected behavior. I will change it though if this goes against the established norm for the package.

Reply:
As alluded to earlier, I don't have a strong enough opinion in this instance to hold up the merge so I can keep the loop. I implemented it the way you requested under the assumption that that is what the average person would prefer.